src/cogent3/__init__.py

```diff
@@ -137,16 +137,22 @@ def make_unaligned_seqs(
     source
         origins of this data, defaults to 'unknown'. Converted to a string
         and added to info["source"].
+    new_type
+        if True, returned SequenceCollection will be of the new type.
+        Support for the old type will be removed as of 2025.6.
     **kw
         other keyword arguments passed to SequenceCollection
     """
+    # refactor: docstring
+    # probably need to add more detail about the new_type flag
+    # should at some point the new_type flag default to true?
```

Comment:
6 months after initial release it will default to true, 12 months it disappears as does the old code